region/brinstar/kraid/Warehouse Energy Tank Room.json

```diff
@@ -124,6 +124,23 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Crystal Flash",
+      "requires": ["h_canCrystalFlash"]
```

Comment:
There is a `enter in ball` way to avoid damage, but otherwise a Beetom attaches instantly.

Reply:
Jumping into the room is also an option. So I think we can say either `canPrepareForNextRoom` or take 2 hits.